src/assets/stylesheets/modules/_react_tabs.scss

```diff
@@ -52,20 +52,15 @@
 
       &[aria-selected=true] {
         color: $tab-color-active;
-        background: $tab-bg-active;
+        background: rgb(68, 90, 182);
```

Comment:
why no variables :( ?

Reply:
because this color is only used in this location. It is never used anywhere else and so does not require a reusable variable